source/main.cc

```diff
@@ -678,12 +678,6 @@ int main (int argc, char *argv[])
         }
       else if (arg=="-j" || arg =="--threads")
         {
-#ifdef ASPECT_USE_PETSC
-          std::cerr << "Using multiple threads (using -j) is not supported when using PETSc for linear algebra. Exiting." << std::endl;
-          return -1;
-#else
-          use_threads = true;
```

Comment:
Actually, you want to keep this line! You want to get rid of the `ifdef` branch because ASPECT no longer supports PETSc, but we still need the `else` branch.